app/assets/stylesheets/barnardos/_checkbox.scss

```diff
@@ -38,7 +38,8 @@ $outer-ring: $inner-ring + ($ring-size * 3);
 }
 
 .checkbox-group__input {
-  display: none;
+  opacity: 0;
+  position: absolute;
```

Comment:
Are we being cautious for older browsers yet?

Reply:
It's a good question. I was rather hoping either you or @ztolley could advise me on the contemporaneous viability of this ploy.

Reply:
(alternative would seem to be a left: -999px or somesuch)

Reply:
We don't know the browser, but we are aiming this at Barnardos stuff so worst case, find out what the Barnardos desktops run.

Where did the fix come from?

Reply:
Me.

Reply:
[Compatibility](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity#Browser_compatibility) doesn't seem bad

Reply:
'k, `left: -9999px` is also perfectly fine, but you lose one small dev advantage – rolling over the 'hidden' `input` in the Inspect mode no longer indicates visually which field we're talking about.

Reply:
As mentioned in the standup, I think we can leave this as is for now, and come back to it if we need to support IE8 as well. That will probably be a separate bigger ticket anyway.